spec/services/post_alerter_spec.rb

```diff
@@ -1101,6 +1101,24 @@ def set_topic_notification_level(user, topic, level_name)
         }.to add_notification(staged_member, :posted)
           .and not_add_notification(staged_non_member, :posted)
       end
+
+      it "does not update existing unread notification" do
+        category = Fabricate(:category)
+        CategoryUser.set_notification_level_for_category(user, CategoryUser.notification_levels[:watching], category.id)
+        topic = Fabricate(:topic, category: category)
+
+        post = Fabricate(:post, topic: topic)
+        PostAlerter.post_created(post)
+        expect(Notification.last.topic_id).to eq(topic.id)
```

Comment:
Isn't `Notification.last` making a SQL request everytime you're calling it? Might be worth storing that in a variable? Especially on the last part of the test where it's called 3 times